src/shapes/IText/__snapshots__/ITextBehavior.test.ts.snap

```diff
@@ -200,23 +157,57 @@ exports[`IText cursor animation snapshot initDelayedCursor true - with NO delay
   "0.536",
   "0.463",
   "0.388",
+  "0.309",
+  "0.228",
+  "0.146",
+  "0.063",
+  "0.000",
+  "1.000",
+  "0.000",
+  "0.001",
+  "0.004",
+  "0.008",
+  "0.014",
+  "0.022",
+  "0.031",
+  "0.043",
+  "0.056",
+  "0.070",
+  "0.086",
+  "0.104",
+  "0.124",
+  "0.145",
+  "0.167",
+  "0.191",
+  "0.216",
+  "0.243",
+  "0.271",
+  "0.300",
+  "0.331",
+  "0.363",
+  "0.395",
+  "0.429",
+  "0.464",
+  "0.500",
+  "0.537",
+  "0.574",
+  "0.612",
+  "0.651",
+  "0.691",
+  "0.731",
+  "0.772",
+  "0.813",
 ]
 `;
 
 exports[`IText cursor animation snapshot selectionStart/selection end will abort animation 1`] = `
 [
+  "0.000",
   "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
+  "0.996",
```

Comment:
Those are the first 160ms. there are 100 of delay that aren't pictured because the animation is not starting., so this is 100 + around 5 ticks of animation